### PR TITLE
(merge #1 for components) final fixes to code in practice components

### PR DESCRIPTION
**Pull request by torres10luis**

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -1,10 +1,14 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css']
 })
-export class AppComponent {
+export class AppComponent implements OnInit {
```
Comment (pratimasakinala) [inline src/app/app.component.ts]: OnInIt is not necessary for all components

File: src/app/danger-message/danger-message.component.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  // tslint:disable-next-line:component-selector
+  selector: 'app-danger-message',
```
Comment (pratimasakinala) [inline src/app/danger-message/danger-message.component.ts]: use class selector here instead of default selector type

**Review by pratimasakinala — CHANGES_REQUESTED** [review]
add some styles to danger and warning components to that they are applied. there are some other comments that i have added. take a look at them

File: src/app/danger-message/danger-message.component.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  // tslint:disable-next-line:component-selector
+  selector: './app-danger-message',
```
Comment (pratimasakinala) [inline src/app/danger-message/danger-message.component.ts]: check if by adding this, you are able to run (ng serve) the application

File: src/app/warning-message/warning-message.component.ts [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-warning-message',
+  template: `
+    <h3>my warning message is successful </h3>
+  `,
+  styles: [`
```
Comment (pratimasakinala) [inline src/app/warning-message/warning-message.component.ts]: expected to use external styles according to instructions

File: src/app/app.component.ts [hunk 5]
```diff
@@ -1,4 +1,4 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
```
Comment (pratimasakinala) [inline src/app/app.component.ts]: no need for the OnInit import as well

File: src/app/app.component.html [hunk 6]
```diff
@@ -1,20 +1,6 @@
...
-    <h2><a target="_blank" rel="noopener" href="https://github.com/angular/angular-cli/wiki">CLI Documentation</a></h2>
-  </li>
-  <li>
-    <h2><a target="_blank" rel="noopener" href="https://blog.angular.io/">Angular blog</a></h2>
-  </li>
-</ul>
 
+<div app-success-message></div>
+
+<app-danger-message></app-danger-message>
```
Comment (pratimasakinala) [inline src/app/app.component.html]: does this work? please check by running ng serve

**Review by pratimasakinala — CHANGES_REQUESTED** [review]
more comments

File: src/app/app.component.html (hunk 6 above)
Comment (torres10luis) [reply src/app/app.component.html]: yes it works

**Review by pratimasakinala — APPROVED** [review]
looks good

**Pull request merged**